docs/bastion.md

```diff
@@ -2,61 +2,84 @@
 ```
-topology:
-    bastion:
-        MachineType: c4.large
+kops create cluster --topology private --networking $provider --bastion $NAME
 ```
-[More information](https://aws.amazon.com/ec2/instance-types/)
 
+### Configure the bastion instance group
 
-### Turn on/off bastion
+You can edit the bastion instance group to make changes. By default the name of the bastion instance group will be `bastions` and you can specify the name of the cluster with `--name` as in:
 
-To turn on/off bastion host setup completely.
-- **Defaults** to `false` if the topology selected is `public`
-- **Defaults** to `true` if the topology selected is `private`
-- **Configure:**
 ```
-kops create cluster --bastion=[true|false]
+kops edit ig bastions --name $KOPS_NAME
 ```
-OR using `kops edit cluster`
+
+You should now be able to edit and configure your bastion instance group.
+
 ```
-topology:
-    bastion:
-        Enable: true
+apiVersion: kops/v1alpha2
+kind: InstanceGroup
+metadata:
+  creationTimestamp: "2017-01-05T13:37:07Z"
+  name: bastions
+spec:
+  associatePublicIp: true
+  image: kope.io/k8s-1.4-debian-jessie-amd64-hvm-ebs-2016-10-21
+  machineType: t2.micro
+  maxSize: 1
+  minSize: 1
+  role: Bastion
+  subnets:
+  - utility-us-east-2a
 ```
 
-### Reach bastion from outside of vpc using a name
+**Note**: If you want to turn off the bastion server, you must set the instance group `maxSize` and `minSize` fields to `0`.
+
+If you do not want the bastion instance group created at all, simply drop the `--bastion` flag off of your create command. The instance group will never be created.
+
+
+### Using a public CNAME to access your bastion
+
+By default the bastion instance group will create a public CNAME alias that will point to the bastion ELB. 
+
+The default bastion name is `bastion-$NAME` as in
 
-- **Default:** CNAME for the bastion is only created when the user explicitly define it using `kops edit cluster`
-- **Configure:** Bastion friendly CNAME can be configured using `kops edit cluster`
 ```
-topology:
+bastion-example.kubernetes.com
+```
+
+Unless a user is using `--dns-zone` which will inherently use the `basion-$ZONE` syntax.
+
+You can define a custom bastion CNAME by editing the main cluster config `kops edit cluster $NAME` and modifying the following block
+
+```
+spec:
+  topology:
     bastion:
-        PublicName: jumper
+      bastionPublicName: bastion-example.kubernetes.com
 ```
 
-### High idle timeout for bastion ASG's ELB. (Configurable LoadBalancer Attributes)
 
-By default, elastic load balancing sets the idle timeout to `60` seconds.
-- **Default:** Bastion ELB in kops will have `120` seconds as their default timeout.
-- **Configure:** This value can be configured using `kops edit cluster`
+### Changing your ELB idle timeout
+
+The bastion is accessed via an AWS ELB. Kops will by default set the bastion ELB idle timeout to 5 minutes. This is important for SSH connections to the bastion that you plan to keep open.
```

Comment:
I would add a note that we need to have an ELB, because the bastion is in an Auto Scale Group.